src/components/card/card.scss

```
@@ -39,6 +40,20 @@ $_cover-gap: ms.step(1, 1rem);
 
 $_focus-overflow: (size.$edge-large * -1);
 
+/// The cloudy modifier adds clouds to the bottom right corner and also applies
+/// some fluid padding to accommodate. The clouds feel best when they are sized
+/// just a _tad_ larger than this fluid padding.
+///
+/// @type Number
+/// @access private
+
+$_cloud-height: fluid.fluid-clamp(
```

Could we rename this variable to better reflect that it's being used for the `background-size` property, rather than a height, which I would expect to contain a single number? Perhaps something like `$_cloud-bg-size`?

```suggestion
$_cloud-bg-size: fluid.fluid-clamp(
```

---

Actually, now i'm super confused because I see you're using the same variable for `block-size`.

I don't understand how we can pass a comma-separated variable into `block-size`?

It's rendering fine in the browser, but that doesn't feel right to me, unless I'm missing something clever here.

---

Oh FFS.

I'm sorry, my bad — I misread this and didn't realize the output of this _IS_ a single value, the numbers you're setting are not a comma-separated list, they're the parameters for the clamp function. Sorry!

---

> I don't understand how we can pass a comma-separated variable into block-size?

I don't think we are?

We're setting the variable to this:

```
fluid.fluid-clamp($min, $max, $min-width, $max-width)
```

Which is outputting this:

```
clamp(min, val, max)
```

Both the Sass function and `clamp` functions accept multiple arguments, but `clamp` represents a single value. Inspecting the preview, I only see one `clamp` in the `block-size`.